### PR TITLE
FAQ updated to include SLOs

### PR DESCRIPTION
**Pull request by hayleycd**
@andrewpollock please confirm that I represented the SLOs correctly. Please feel free to suggest changes if you want to adjust the wording slightly.

### CODE REVIEW

File: gcp/appengine/frontend3/src/templates/faq.html [hunk 1]
```diff
@@ -91,6 +91,15 @@ <h3 class="question">Is the database available to download?</h3>
           <p>The database in available in a GCS bucket maintained by OSV: <a href="https://osv-vulnerabilities.storage.googleapis.com">gs://osv-vulnerabilities</a></p>
           <p>More information about how to download the database is available <a href="https://github.com/google/osv.dev#data-dumps">here.</a></p>
         </div>
+        <h3 class="question">What are OSV's service level objectives (SLOs)?</h3>
+        <div class="answer">
+          <p>OSV strives to provide reliable vulnerability information to our users. To support that goal, we work to meet the following service level objectives:</p>
```
Comment (andrewpollock) [inline gcp/appengine/frontend3/src/templates/faq.html:96]:
```suggestion
          <p>OSV strives to provide reliable vulnerability information to our users. To support that goal, target the following service level objectives:</p>
```

**Review by andrewpollock — APPROVED** [review]
LGTM, minor suggested wording tweak

**Comment by hayleycd** [comment]
Sounds good to me!

**Pull request merged**